cli/mender-artifact/modify.go

```diff
@@ -61,7 +61,7 @@ func modifyArtifact(c *cli.Context) error {
 			defer os.Remove(mc.path)
 		}
 	} else if len(modifyCandidates) == 4 { // sdimg
-		modifyCandidates = modifyCandidates[1:2]
+		modifyCandidates = modifyCandidates[1:3]
```

Comment:
I don't get it. Are we simply including rootfsB in the modification, and that's it? Correct me if I'm wrong, but this doesn't sound like it would affect vanilla images, which is the problem described in [MEN-2294](https://tracker.mender.io/browse/MEN-2294). Can you clarify?

Reply:
The problem was that the sdimg was not repacked due to the array length being too short, for the `if` repack test below. What do you mean by Vanilla-imags? Ticket says that only sdimg's are effected.